chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -1046,7 +1046,8 @@ Example
 Mapbox Vector Tiles
 -------------------
 A RESTFul implementation of `Mapbox Vector Tiles <https://www.mapbox.com/vector-tiles>`_.
-For the testing phase, the service is free for use. See conditions `conditions <https://www.swisstopo.ch>`_ give us feedback via our `survey <https://findmind.ch/c/vectortilesEN>`_  
+The service has be developed for testing purposes. The service is free for use during the test phase (until June 30th 2019). swisstopo reserves the right to change or suspend the service at any time without prior notice.
+Feel free to test the service and give us feedback to help use optimize our product via our `survey <https://findmind.ch/c/vectortilesEN>`_  
 
```

Comment:
@davidoesch there is a typo:

"... to help use..." i guess should be "... to help us..:"

Reply:
thx fixed